helm/kyverno-policies-connectivity/values.yaml

```diff
@@ -6,6 +6,12 @@ certManager:
 proxy:
   enabled: false
 
+mimir:
+  enabled: false
+  prometheus:
+    name: mimir-to-grafana-cloud
+    namespace: mimir
```

Comment:
I think you don't need namespace or you maybe want to use it?

For the circleci, I htink you need to run make generate

Reply:
> I think you don't need namespace or you maybe want to use it?

I don't think it's needed either so let's just remove it from the values.

> For the circleci, I htink you need to run make generate

Ok let's try that then :)

Reply:
you actually need to write the policies [here](https://github.com/giantswarm/kyverno-policies-connectivity/tree/main/policies/connectivity/proxy) and run `make`. the ones in the `helm` dir are generated.

don't ask me why 😆

Reply:
Yeah I just figured out this and thought it's really weird :sweat_smile: